spec/model/user_spec.rb

```diff
@@ -0,0 +1,184 @@
+require "rails_helper"
+include ApplicationHelper
+RSpec.describe User, type: :model do
+  describe "Associations" do
+    it do
+      is_expected.to have_many(:messages).with_foreign_key(:user_send_id).dependent(:destroy)
+      is_expected.to have_many(:active_relationships).with_foreign_key(:follower_id).dependent(:destroy)
+      is_expected.to have_many(:passive_relationships).with_foreign_key(:followed_id).dependent(:destroy)
+    end
+  end
+
+  describe "Validations" do
```

Comment:
Describle, context, it có tác dụng gì em

Reply:
describe và context chia thành các khối và coi như là 1 test case anh ạ, describe thì mô tả method đó, còn context sẽ mô tả các trường hợp xảy ra với method đó, với mỗi trường hợp xảy ra thì sử dụng it trong đó.